### DISCUSSION TITLE
can-you-use-the-dotnet-8-identity-api-endpoints-with-identityserver/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#390] [Announcements]
# can-you-use-the-dotnet-8-identity-api-endpoints-with-identityserver/

In this post I discuss how the Identity endpoints interact and relate to OpenID Connect implementations, and whether you can (or should) combine them.

https://andrewlock.net/can-you-use-the-dotnet-8-identity-api-endpoints-with-identityserver/

**Comment by ShaunCurtis** [comment]
Thanks.  I read your three articles on the subject with interest.   I agree with what you say: it was good to get affirmation of my concerns,  I'm updating an application [to Net8] I wrote in the early days of Blazor.  I'm plagiarizing and customizing the server side pages I need.  I don't like the uncontrolled register action either.
